script.js

```diff
@@ -6,4 +9,317 @@ function openNav(){
+
+    function getCurrCheckedSeason($param) {
+        var currCheckedSeason = $param.val();
+        if (typeof(currCheckedSeason) !== "undefined")
+        {
+            return currCheckedSeason;
+        }
+    }
+
+    //if hits enter during season text box, trigger button
+    $('#newSeasonInputBox').keypress(function(e){
+        if (e.keyCode == 13)
+        {
+            addNewSeason();
+        }
+    });
+
+    //if hits enter while entering points, trigger save button
+    $('#elementPoints').keypress(function(e){
+        if (e.keyCode == 13)
+        {
+            $('#scoringSaveButton').click();
+        }
+    });
+
+    //adds new element to table, handles things like element already exists and season not checked
+    $("#scoringSaveButton").click(function(){
+        var newElementName = $("#elementKey").val().trim().toString();
+        var newElementPoints = $("#elementPoints").val().trim();
+        
+        if (newElementName !== "" || newElementPoints !== "")
+        {
+            var season = $("input[name=seasonListItem]:checked","#seasonRadioList").val();
+            if (typeof(season) !== "undefined")
+            {
+                var cookieName = '/season_config/' + season + '/' + newElementName;
+                var cookieValue = newElementPoints;
+                
+                var cookieList = document.cookie.split(";");
+                var elementAlreadyStored = false;
+                for (var i = 0; i < cookieList.length;i++)
+                {
+                    if (cookieList[i].trim().startsWith('/season_config/' + season))
+                    {
+                        var str = cookieList[i];
+                        var checkerName = str.substring(str.lastIndexOf('/')+1,str.indexOf('='));
+                        if (checkerName == newElementName)
+                        {
+                            elementAlreadyStored = true;
+                            break;
+                        }
+                    }
+                }
+                
+                if (!elementAlreadyStored)
+                {
+                    $("#elementConfirmationBox").text(" ");
+                    document.cookie = cookieName + "=" + cookieValue + 
+                    "; expires=Wed, 1 Jan 2038 12:00:00 UTC; path=/";
+                    //doc.cookie --> /season_config/season/elementName=elementValue;...
+
+                    var newTRHtml = "<tr><td>" + newElementName + 
+                    "</td><td>" + newElementPoints + "</td></tr>";
+                    
+                    $("#scoringTable").append(newTRHtml);
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" added.<\i>');
+
+                    $("#elementKey").val("");
+                    $("#elementPoints").val("");
+                } else
+                {
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" already exists.<\i>');
+                }
```

Comment:
What if I want to change it?

Probably just making it always over-write works better.

Reply:
might also want to think of a way to delete - but a current way to do that would be to delete the cookie, so that's a lower priority.